src/ExEss/Cms/Exception/NotAuthenticatedException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace ExEss\Cms\Exception;
+
+class NotAuthenticatedException extends \DomainException
```

Comment:
shouldn't we extend something like vendor/symfony/security-core/Exception/AuthenticationException.php ? or just use that one

Reply:
ah wait not, that the AuthenticationException, not NotAuthenticated